phys2bids/tests/test_integration.py

```diff
@@ -161,66 +158,125 @@ def test_integration_multifreq(multifreq_acq_file):
     # Check sampling frequency
-    assert check_string(log_info, 'Sampling Frequency', '625.0')
+    assert check_string(log_info, 'Sampling Frequency', '40.0')
     # Check sampling frequency
-    assert check_string(log_info, 'Sampling started', '0.2905')
+    assert check_string(log_info, 'Sampling started', '-157.8535')
     # Check start time
     assert check_string(log_info, 'first trigger', 'Time 0', is_num=False)
 
     # Checks json file
-    with open(os.path.join(test_path, 'Test_belt_pulse_multifreq_625.0.json')) as json_file:
+    with open(os.path.join(test_path, 'Test1_multifreq_onescan_40.0.json')) as json_file:
         json_data = json.load(json_file)
 
     # Compares values in json file with ground truth
-    assert math.isclose(json_data['SamplingFrequency'], 625.0)
-    assert math.isclose(json_data['StartTime'], 0.2905)
-    assert json_data['Columns'] == ['PULSE - Custom, DA100C']
+    assert math.isclose(json_data['SamplingFrequency'], 40.0)
+    assert math.isclose(json_data['StartTime'], -157.8535)
+    assert json_data['Columns'] == ['O2']
 
     """
-    Checks 10000 Hz output
+    Checks 100 Hz output
     """
     # Read log file of frequency 625 (note that this file is not the logger file)
-    with open(os.path.join(test_path, 'Test_belt_pulse_multifreq_10000.0.log')) as log_info:
+    with open(os.path.join(test_path, 'Test1_multifreq_onescan_100.0.log')) as log_info:
         log_info = log_info.readlines()
 
     # Check timepoints expected
     assert check_string(log_info, 'Timepoints expected', '1')
     # Check timepoints found
-    assert check_string(log_info, 'Timepoints found', '60')
+    assert check_string(log_info, 'Timepoints found', '0')
     # Check sampling frequency
-    assert check_string(log_info, 'Sampling Frequency', '10000.0')
+    assert check_string(log_info, 'Sampling Frequency', '100.0')
     # Check sampling started
-    assert check_string(log_info, 'Sampling started', '10.4251')
+    assert check_string(log_info, 'Sampling started', '-0.3057')
     # Check start time
     assert check_string(log_info, 'first trigger', 'Time 0', is_num=False)
 
     # Checks json file
-    with open(os.path.join(test_path, 'Test_belt_pulse_multifreq_10000.0.json')) as json_file:
+    with open(os.path.join(test_path, 'Test1_multifreq_onescan_100.0.json')) as json_file:
         json_data = json.load(json_file)
 
     # Compares values in json file with ground truth
-    assert math.isclose(json_data['SamplingFrequency'], 10000.0)
-    assert math.isclose(json_data['StartTime'], 10.4251)
-    assert json_data['Columns'] == ['time', 'RESP - RSP100C',
-                                    'MR TRIGGER - Custom, HLT100C - A 5', 'PPG100C', 'CO2', 'O2']
+    assert math.isclose(json_data['SamplingFrequency'], 100.0)
+    assert math.isclose(json_data['StartTime'], -0.3057)
+    assert json_data['Columns'] == ['CO2']
+
+    """
+    Checks 100 Hz output
```

Comment:
Is this next section checking the 100 Hz or 500 Hz output?

Reply:
Thanks @rmarkello ! Copy-pasting typo 😅